test/index.js

```diff
@@ -93,6 +93,11 @@ test('works with external stylesheets', async t => {
   t.snapshot(code)
 })
 
+test('works with external stylesheets (prettier)', async t => {
+  const { code } = await transform('./fixtures/external-stylesheet-prettier.js')
+  t.snapshot(code)
```

Comment:
I guess that you can merge this with the existing test for external styles. If you want to keep them separate then maybe we should rename it because as I said early the bug it is not about prettier